### DISCUSSION TITLE
Problem with Scarf Seam inner walls vanishing when using MM?

### DISCUSSION THREAD

**Discussion by Toffypops** [OrcaSlicer/OrcaSlicer#5180] [General]
https://youtu.be/QCZzD_Lv-NY

Made a video showing it.

**Comment by vgdh** [comment]
what you have changed is seam gap. That is the distance between the start and the end of the loop. It's not suppose to be as big value as 20mm.

**Reply by Toffypops** [reply]
Should be blocked to go that high then. Prevents a lot of confusion. :)

**Reply by vgdh** [reply]
I agree
